simpa/core/simulation_modules/reconstruction_module/reconstruction_utils.py

```diff
@@ -481,7 +484,7 @@ def compute_for_one_dimension(start_in_mm: float, end_in_mm: float) -> Tuple[int
         start_temp = start_in_mm / spacing_in_mm
         end_temp = end_in_mm / spacing_in_mm
         dim_temp = np.abs(end_temp - start_temp)
-        dim = int(np.round(dim_temp))
+        dim = round_away_from_zero(dim_temp)
```

Comment:
This change only affects cases where `dim_temp` is an X.5 value. As the rounding difference (`diff`) is anyway compensated for in lines 489 and 490, `start` and `end` should only be affected in edge cases and there the maximal deviation should amount to the same as before but in the different direction. For `dim` it might, however, make an off-by-one difference, if e.g. 18.5 will be rounded to 18 or 19 so here the change to `round_away from_zero` makes sense 👍